mteb/tasks/BitextMining/multilingual/IndicGenBenchFloresBitextMining.py

```diff
@@ -0,0 +1,171 @@
+from __future__ import annotations
+
+from typing import Any
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskBitextMining, CrosslingualTask
+
+_LANGUAGES = [
+    "ben-Beng",
+    "guj_Gujr",
+    "hin_Deva",
+    "kan_Knda",
+    "mal_Mlym",
+    "mar_Deva",
+    "tam_Taml",
+    "tel_Telu",
+    "urd_Arab",
+    "asm_Beng",
+    "bho_Deva",
+    "nep_Deva",
+    "ory_Orya",
+    "pan_Guru",
+    "pus_Arab",
+    "san-Deva",
+    "awa_Deva",
+    "bgc_Deva",
+    "bod_Tibt",
+    "boy_Deva",
+    "gbm_Deva",
+    "gom_Deva",
+    "hne_Deva",
+    "raj_Deva",
+    "mai_Deva",
+    "mni_Mtei",
+    "mup_Deva",
+    "mwr_Deva",
+    "sat_Olck",
+]
```

Comment:
Inconsistency due to - vs _
Can you use `-` everywhere? AFAIK that's the correct way to represent.

Ref: `ISO_LANGUAGE_SCRIPT = str  # a 3-letter ISO 639-3 language code followed by a 4-letter ISO 15924 script code (e.g. "eng-Latn")`

Reply:
@KranthiGV Basically I copied this for ease but the ones getting passed to the Task Metadata are transformed using get_lang_pairs to be in format similar to tatoeba(https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/TatoebaBitextMining.py) and a similar operation has also been performed in FloresBitext class (https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/BitextMining/multilingual/FloresBitextMining.py#L220)
![Screenshot 2024-05-16 at 12 32 13 PM](https://github.com/embeddings-benchmark/mteb/assets/53102161/ccf61fe4-b31f-4cce-b586-24edac6caa1a)

Reply:
It's good to be consistent. I noticed that some existing datasets, such as `mteb/tasks/BitextMining/multilingual/FloresBitextMining.py` has underscores and uses `-` to signify the cross lingual component. @KennethEnevoldsen or @imenelydiaker what's the preferred way here?

Reply:
@isaac-chung btw the input to the eval_langs param of TaskMetadata is consistent and does not use underscores in all existing datasets the underscores are replaced by '-' and follow pattern similar to Tatoeba(dictionary mapping huggingface subsets to lists of languages), 
![Screenshot 2024-05-16 at 2 38 44 PM](https://github.com/embeddings-benchmark/mteb/assets/53102161/d9c98006-1212-49a7-baf6-bf329b44cfd8)

![Screenshot 2024-05-16 at 2 37 50 PM](https://github.com/embeddings-benchmark/mteb/assets/53102161/623006f9-3516-4fbe-9764-ff2b2ec97209)

I have followed the same pattern.

Reply:
Oh gotcha. I had missed your earlier comment. Thanks!